ext/storage_sources/gcp_store/gcp_storage_source.cpp

```diff
@@ -291,89 +310,136 @@ gcp_file_open(WT_FILE_SYSTEM *file_system, WT_SESSION *session, const char *name
     return 0;
 }
 
+// POSIX remove, not supported for cloud objects.
```

Comment:
I'm a bit confused. Should this say `POSIX remove is not supported for cloud objects.`?

Reply:
It should say that, thanks Tammy!
updated d11e042e49654d91f7ee8bd8450515ee77e1766e